crud/helpers.go

```diff
@@ -136,14 +138,26 @@ func LoadBalancerResourceGet(s BaseCrud, workReq *baremetal.WorkRequest) (id str
 	return id, false, nil
 }
 
+func sleepHelper(retryNum uint) {
+	secondsToSleep := 1 << retryNum
+	log.Printf("[DEBUG] Got a retriable error. Waiting %d seconds and trying again...", secondsToSleep)
+	time.Sleep(time.Duration(secondsToSleep) * time.Second)
+}
+
+func isRetriableError(error string) bool {
+	match, _ := regexp.MatchString("status\\s*:\\s*40[134]", strings.ToLower(error))
```

Comment:
We consider a 400 a retryable error, but not a 404? Are we sure none of the cases being hit where retries were necessary returned 404s?